fe2-android/app/src/main/java/com/github/dedis/student20_pop/utility/json/JsonCloseRollCallSerializer.java

```diff
@@ -0,0 +1,33 @@
+package com.github.dedis.student20_pop.utility.json;
```

Comment:
I don't think those serializers are needed. I might be mistaking, but from my understanding, as your data objects only have json serializable attributes, you can directly serialize them using gson, and there is no need to pass via a custom serializer.

Reply:
You are right it was mainly about converting fields such as "openedAt" to "opened_at" but I found a way using gson annotations and I was also worried about lists. I removed those classes it seems to work fine as well.

Reply:
If the classes are not used and the serialization/deserialization works fine without them, then you can delete them